### ISSUE TITLE
Error when using Vite

### ISSUE THREAD

**Issue by tyrw** [userfront/userfront-vue#3]
From a customer:

`Vue 3.2.19 and Vite 2.6.2`

We are having issues with SSO, Vue, and Vite. 
Here is the error we are getting: 

```
> node_modules/@userfront/vue/build/userfront-vue.module.js:1:7: error: No matching export in "node_modules/vue/dist/vue.runtime.esm-bundler.js" for import "default"
1 │ import e from"vue";import r from"@anymod/core";import t from"@userfront/core";const o=function(){try{return n.isScript1Loading?Promise.resolve():Pr...
╵ ^

[vite] error while updating dependencies:
Error: Build failed with 1 error:
node_modules/@userfront/vue/build/userfront-vue.module.js:1:7: error: No matching export in "node_modules/vue/dist/vue.runtime.esm-bundler.js" for import "default"
```

**Comment by gemmadlou** [comment]
Hi @tyrw I've done some work on a fork for Vue3 compatibility - but of course, I wouldn't want to create a PR that breaks Vue 2 compatibility. I'll continue to use my one as I need it for a project I'm working on.

https://github.com/gemmadlou/userfront-vue

I'm happy to make a PR to an appropriate branch and fix any tests I've broken during the upgrade.

**Comment by tyrw** [comment]
Thanks @gemmadlou! This is great, and we'll reference it when we update everything to work with both Vue 2 and Vue 3!

**Comment by RJFelix** [comment]
Should be fixed via #9
